packages/fast-components/src/name-tag/name-tag.styles.ts

```diff
@@ -1,6 +1,10 @@
 import { css } from "@microsoft/fast-element";
 
 export const NameTagStyles = css`

```

Comment:
Looks like you have another PR to simply remove this component - should we just close this one?

Reply:
We can - I was simply removing the export in the other; should we drop the example all up?

Reply:
Oh sorry - I thought it removed everything. That's fine - we can remove the actual component when we get a few components through.